generators/php/codegen/src/asIs/JsonDecoder.Template.php

```diff
@@ -106,14 +106,25 @@ public static function decodeInt(string $json): int {
      * @return mixed[]|array<string, mixed> The deserialized array.
      * @throws JsonException If the decoded value is not an array.
      */
-    public static function jsonToDeserializedArray(string $json, array $type): array {
+    public static function decodeArray(string $json, array $type): array {
         $decoded = self::decode($json);
         if (!is_array($decoded)) {
             throw new JsonException("Unexpected non-array json value: " . $json);
         }
         return JsonDeserializer::deserializeArray($decoded, $type);
     }
 
+    /**
+     * Decodes a JSON string and returns a mixed.
+     *
+     * @param string $json The JSON string to decode.
+     * @return mixed The decoded mixed.
+     * @throws JsonException If the decoded value is not an mixed.
+     */
+    public static function decodeMixed(string $json): mixed {
+        return self::decode($json);
+    }
```

Comment:
should we just replace the `decode` method?

Reply:
I actually kind of like having both so that everything else just reads as `decode`, but I could be convinced either way. If you'd rather just have `decodeMixed`, I'm OK with that